pkg/config/application.go

```diff
@@ -519,6 +527,25 @@ func (e *SecretEncryption) Validate() error {
 	return nil
 }
 
+type Attachment struct {
+	// List of files which contain embedding source data.
+	Sources map[string]string `json:"sources"`
```

Comment:
It's possible to understand necessary to specify the source path here, but it might be hard to get why this field is map instead of slice just from this comment.

Reply:
Updated, thank you 👍